public/service-worker.js

```diff
@@ -1,68 +1,11 @@
-importScripts('sw-lib.v0.0.16.min.js');
-importScripts('precache-manifest.js');
+importScripts('https://unpkg.com/workbox-sw@0.0.2');
```

Comment:
This doesn't seem to work in Chrome?
I'm getting error when Sw tried loading it;
`The script resource is behind a redirect, which is disallowed.`